Gemfile.lock

```diff
@@ -1,28 +1,26 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    about_yml (0.0.3)
```

Comment:
The motivation for this upgrade is that this version of the `about_yml` gem seems to have been yanked.

https://rubygems.org/gems/about_yml

/cc @mbland